tests/Handler/AddPrefixCommandHandlerTest.php

```diff
@@ -94,9 +94,9 @@ public function testAddPrefixToDirectory()
         $args = self::$command->parseArgs(new StringArgs('MyPrefix\\\\ dir'.DIRECTORY_SEPARATOR.'dir'));
 
         $expected = <<<EOF
-Scoping $this->tempDir/dir/dir/MyClass.php. . . Success
 Scoping $this->tempDir/dir/dir/MySecondClass.php. . . Success
 Scoping $this->tempDir/dir/dir/MyThirdClass.php. . . Success
+Scoping $this->tempDir/dir/dir/MyClass.php. . . Success
```

Comment:
Order of MyClass line is acting up. Passes on 5.6-7.1 with/without xdebug locally. Fails on all builds with Travis.